.github/renovate.json

```
@@ -0,0 +1,20 @@
+{
```

A little editor nicety:

```suggestion
{
	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
```